.storybook/main.js

```diff
@@ -36,7 +36,7 @@ const addVanillaExtractSupport = (config) => {
         }
     });
 
-    config.plugins.push(new VanillaExtractPlugin(), new MiniCssExtractPlugin());
+    config.plugins.push(new VanillaExtractPlugin(), new MiniCssExtractPlugin({ignoreOrder: true}));
```

Comment:
what does this change do?

Reply:
Basically, ignore warnings about order of imports in bundles. Without it, there are a lot of warnings when building the storybook:

![image](https://github.com/Telefonica/mistica-web/assets/25785151/f0a1fbfe-78da-43d2-9245-ea9bf9de25b0)

Reply:
hmmm... and why do we have those warnings?

Reply:
Discussed offline. We believe it's safe to ignore these warnings for the storybook. I've added a comment in the code providing a link where there is an explanation of the reason this happens ([Link](https://stackoverflow.com/questions/51971857/mini-css-extract-plugin-warning-in-chunk-chunkname-mini-css-extract-plugin-con/67579319#67579319))